drivers/SmartThings/wemo/src/parser.lua

```diff
@@ -30,49 +30,41 @@ end
 
 local function handle_binary_state(device, value)
   -- parses wemo insight style ("8|1611850428|58|...") state, works just fine for a single value too
```

Comment:
Might be nice if there was a link to a doc that describes what the format being parsed looks like but I'm assuming that document doesn't exist.

Reply:
wemo? documentation? I wish. These numbers are still a mystery to me.